packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -1101,17 +1097,17 @@ class TableRow<T> extends StatefulWidget {
         activeSearchMatchNotifier = null,
         super(key: key);
 
-  final LinkedScrollControllerGroup linkedScrollControllerGroup;
+  final LinkedScrollControllerGroup? linkedScrollControllerGroup;
```

Comment:
non-nullable. A quick tip would be to check the call sites for a constructor or a method to verify whether something needs to be nullable.

Reply:
thanks. done